src/lib/netlify-auth.js

```diff
@@ -74,6 +74,25 @@ class Authenticator {
     return host === 'localhost' ? null : host;
   }
 
+  //NOTE the auth server needs to have CORS enabled for this direct request to work
+  //todo: is this the best place for this refresh auth token code to live?
+  //todo: will netlify auth server use cors and direct request?
+  refreshAuthToken(options) {
+    // const base_url = this.base_url; //todo: how to get proper config base_url from app/props config.base_url
+    const base_url = "https://netlify-bitbucket-oauth.now.sh";
```

Comment:
Can you just use `this.base_url` here like the other functions do (created at line 34)?